tests/testthat/run-dbi-tests.R

```diff
@@ -153,6 +154,21 @@ test_that("dbWriteTable skips importing empty dataframe", {
   dbDisconnect(con)
 })
 
+test_that("dbReadTable", {
```

Comment:
we did not have a test for it? ok?

Reply:
No, we had only some tests for dbWriteTable().
I found it important to have at least one test for dbReadTable().